desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -3161,49 +3158,28 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
         };
         function upload() {
           var fd = new FormData();
-          fd.append('fileFormat', ko.mapping.toJSON(viewModel.createWizard.source));
+          var files = $('#inputfile')[0].files[0];
+          fd.append('inputfile', files);
+          var file_size = files.size;
+          if (file_size < 30*1000) {
             $.ajax({
-              url:"/indexer/api/indexer/guess_format",
+              url:"/indexer/api/indexer/upload_local_file",
               type: 'post',
               data: fd,
               contentType:false,
               cache: false,
               processData:false,
-              beforeSend: function (){
-                viewModel.createWizard.isGuessingFormat(true);
-              },
-              success:function (resp) {
-                var newFormat = ko.mapping.fromJS(new FileType(resp['type'], resp));
-                viewModel.createWizard.source.format(newFormat);
-                viewModel.createWizard.isGuessingFormat(false);
+              success:function (response) {
+                viewModel.createWizard.source.path(response['file_url']);
               }
+            
             });
+          }
+          else {
+            alert("As of now File size must be less than 30KB");
```

Comment:
FYI: we don't use alert but `$.jHueNotify.`

We can investigate the size limit in follow-up (should be more)